### ISSUE TITLE
sudo ua enable cc-eal --beta errors out with GPG key '/usr/share/keyrings/ubuntu-cc-keyring.gpg' not found

### ISSUE THREAD

**Issue by davecore82** [canonical/ubuntu-advantage-client#1527]
Using Ubuntu Pro 16.04, trying to run the beta command `sudo ua enable cc-eal --beta` errors out with the following error:

```
ubuntu@xenialpro:~$ sudo ua enable cc-eal --beta
One moment, checking your subscription first
GPG key '/usr/share/keyrings/ubuntu-cc-keyring.gpg' not found
```

This is with the UA client version 26.2~16.04.1.

Is common criteria even supposed to work in the latest Ubuntu Pro images? I know that CC requires precisely Ubuntu 16.04.**4**, so I don't think this would work anyway considering that we're already at version 16.04.7 in the latest Ubuntu Pro images.

I think we should either allow common criteria to work on 16.04.7 and then make `sudo ua enable cc-eal --beta` work, or we should remove cc-eal completely from the UA client. If we remove it, we should update our Ubuntu Pro documentation/datasheets/etc. because I don't see how we could let customers use common criteria on Ubuntu Pro today.

**Comment by davecore82** [comment]
I filed a similar bug on Launchpad under the security certifications project: 

https://bugs.launchpad.net/ubuntu-security-certifications/+bug/1922794

**Comment by renanrodrigo** [comment]
Closing this bug as fixed on the UA side on release `27.4`.
About making it available to other Ubuntu minor releases, I'd recommend a stronger follow up with the security team.
Feel free to reach out in case of any doubt.
